bagging.md

```diff
@@ -1,22 +1,32 @@
 # Baggers
 
 - Getting set up as a Bagger
-  - Talk to your DataRescue guide to make sure you can become a Bagger.
+  - Apply to become a Bagger 
+    - By asking your DataRescue guide or by filling out [this form](XXX)
```

Comment:
Can we remove the links for now then? I'd rather not have dead links in documentation